src/components/DateRangePicker.jsx

```diff
@@ -247,13 +255,28 @@ class DateRangePicker extends React.PureComponent {
   }
 
   setDayPickerContainerRef(ref) {
+    if (this.dayPickerContainer) this.removeDayPickerEventListeners();
+    if (!ref) return;
     this.dayPickerContainer = ref;
+    this.addDayPickerEventListeners();
```

Comment:
Is it expensive to be removing and adding this all the time? Can we add/remove it only if the ref has changed maybe?

Reply:
@majapw I didn't realize that the ref would be `set` unless it changed. Is React not that efficient at this? If it's not then we should definitely be more careful 😺

Reply:
@majapw Is React not efficient about only calling this when the `ref` changes? If not, then we will need to be a bit more careful for sure.

Reply:
I guess I don't actually know. 🤔I know it becomes null sometimes (presumably during rerenders and I guess it would be a new object afterwards), but maybe that would be more of a symptom of unnecessary rerenders. This is probably fine. :)

Reply:
That happens when the ref callback is `!==` across renders, for one.